tests/Oryx.Integration.Tests/NodeEndToEndTests.cs

```diff
@@ -252,14 +251,47 @@ public async Task CanBuildAndRun_NodeApp(string nodeVersion)
                 });
         }
 
+        [Fact]
+        public async Task CanBuildAndRun_NodeApp_WithBuildpack()
+        {
+            // Arrange
+            var appName = "webfrontend";
+            var appVolume = CreateAppVolume(appName);
+            var dockerPort = new DockerVolume("/var/run/docker.sock", "/var/run/docker.sock");
```

Comment:
Curious, what does this do?

Reply:
Allows the `pack` binary to use the Docker daemon